CHANGES.md

```diff
@@ -3,6 +3,10 @@
 
 ## CKEditor 4.9
 
+New Features:
+
+* [#1338](https://github.com/ckeditor/ckeditor-dev/issues/1338): Added function key mappings (F1, F2, etc.). It allows to set keystrokes with function keys. e.g. `editor.setKeystroke(CKEDITOR.CTRL + 117, "Function that triggers on 'Ctrl+F6'");`.
```

Comment:
Try to keep changelog entries simpler, here there's no real need to explain APIs in depth but just inform that labels are now displaying function keys too.